lv_objx/lv_list.c

```diff
@@ -511,13 +511,17 @@ lv_obj_t * lv_list_get_next_btn(const lv_obj_t * list, lv_obj_t * prev_btn)
 
 /**
  * Get the index of the button in the list
- * @param list pointer to a list object
+ * @param list pointer to a list object. If NULL, assumes btn is part of a list.
  * @param btn pointer to a list element (button)
  * @return the index of the button in the list, or -1 of the button not in this list
  */
 int32_t lv_list_get_btn_index(const lv_obj_t * list, const lv_obj_t * btn)
 {
 	int index = 0;
+    if( list == NULL ){
```

Comment:
@kisvegabor I'm debating whether we should separate re-indenting from adding the feature. Do you think it matters?

Reply:
the retab and the feature are separate commits, so if for some reason you need to roll back it'd be quite easy.

Reply:
@embeddedt It would be better to have them in the same commit but we can't avoid having formatting commits.
@BrianPugh Next time please use more descriptive commit message, like `lv_list: fix indentation`

Rewriting the commit message or amending the already pushed commits is quite hacky. So, for now, we can leave it.

Reply:
I'll keep that in mind for the future!